.github/workflows/build_linux_packages.yml

```diff
@@ -136,15 +149,23 @@ jobs:
 
       # TODO: Move to script
       - name: Upload Logs
+        if: always()
         run: |
-          aws s3 cp build/logs/ s3://therock-artifacts/${{github.run_id}}/logs/${{env.AMDGPU_FAMILIES}}/ \
-            --recursive \
-            --content-type="text/plain" \
-            --exclude "*" \
-            --include "*.log"
-          aws s3 cp build/logs/index.html s3://therock-artifacts/${{github.run_id}}/logs/${{env.AMDGPU_FAMILIES}}/
-
+          python3 build_tools/upload_logs_to_s3.py
+        env:
+          GITHUB_RUN_ID: ${{ github.run_id }}
+          AMDGPU_FAMILIES: ${{ env.AMDGPU_FAMILIES }}
+          S3_BUCKET: therock-artifacts
       - name: Add Links to Job Summary
+        if: always()
         run: |
-          echo '* [Artifacts](https://therock-artifacts.s3.us-east-2.amazonaws.com/${{github.run_id}}/index-${{env.AMDGPU_FAMILIES}}.html)' >> $GITHUB_STEP_SUMMARY
-          echo '* [Build logs](https://therock-artifacts.s3.us-east-2.amazonaws.com/${{github.run_id}}/logs/${{env.AMDGPU_FAMILIES}}/index.html)' >> $GITHUB_STEP_SUMMARY
+          LOG_URL="https://s3.amazonaws.com/therock-artifacts/${GITHUB_RUN_ID}/logs/${AMDGPU_FAMILIES}/index.html"
```

Comment:
The bucket URL is invalid:
> The bucket you are attempting to access must be addressed using the specified endpoint. Please send all future requests to this endpoint.

Reply:
It came to my tired moment -  fixing it.

Reply:
Fixed the URL